humble/distribution.yaml

```diff
@@ -6412,6 +6412,20 @@ repositories:
       url: https://github.com/mrpt-ros-pkg/pose_cov_ops.git
       version: master
     status: maintained
+  privacy_msgs:
+    doc:
+      type: git
+      url: https://github.com/ros4hri/privacy_msgs.git
+      version: humble-devel
+    release:
+      tags:
+        release: release/humble/{package}/{version}
+      url: https://github.com/ros4hri/privacy_msgs-release.git
```

Comment:
@severin-lemaignan can we move this release repository to ros2-gbp?

I can start the PR and mirroring process to add it to the ros4hri team: https://github.com/ros2-gbp/ros2-gbp-github-org/blob/latest/ros4hri.tf

Reply:
yes please! (sorry, I always forget how the ros2-gbp process work. It is certainly documented somewhere, but I just don't seem to have the required mental bandwidth to handle it properly 😔)